bip-0340.mediawiki

```diff
@@ -75,27 +75,22 @@ We note that key prefixing is not strictly necessary for transaction signatures
 
 Using the first option would be slightly more efficient for verification (around 10%), but we prioritize compactness, and therefore choose option 3.
 
+Despite halving the size of the set of valid public keys, implicit Y coordinates are not a reduction in security. Informally, if a fast algorithm existed to compute the discrete logarithm of an X-only public key, then it could also be used to compute the discrete logarithm of a full public key: apply it to the X coordinate, and then optionally negate the result. This shows that breaking an X-only public key can be at most a small constant term faster than breaking a full one.<ref>This can be formalized by a simple reduction that reduces an attack on Schnorr signatures with implicit Y coordinates to an attack to Schnorr signatures with explicit Y coordinates. The reduction works by reencoding public keys and negating the result of the hash function, which is modeled as random oracle, whenever the challenge public key has an explicit Y coordinate that is odd. A proof sketch can be found [https://medium.com/blockstream/reducing-bitcoin-transaction-sizes-with-x-only-pubkeys-f86476af05d7 here].</ref>.
```

Comment:
This doesn't seem to be the right place still, because the `'''implicit Y coordinates'''` section starts just below this paragraph. Perhaps move it right below the first sentence of the `implicit Y coordinates` section.

Reply:
You're right that it shouldn't talk about implicit Y coordinates, but on the other hand, the security argument doesn't really rely on that (and would equally apply to a hypothetical scheme where the Y coordinate is allowed to either).

I've dropped the commit for now, and we can discuss phrasing separately. It's unrelated to this change anyway.